app/views/admin/protocol_drugs/show.html.erb

```diff
@@ -18,6 +18,10 @@
   <%= @protocol_drug.protocol %>
 </p>
 
-<% if policy([:manage, @protocol_drug]).edit? %>
+<% if Flipper.enabled?(:new_permissions_system_aug_2020, current_admin) %>
```

Comment:
There might be a bug here. Doesn't this bit mean once we enable the flipper flag, everyone will see a link?

Reply:
_ditto_